### ISSUE TITLE
#-1279386434------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1834]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div class="sc-kRRyDe eWMgrj">`

<strong>Element Location: </strong>.sc-kRRyDe

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-1279386434_All page content should be contained by landmarks.png`
